### PR TITLE
Publish logins AAR to nalexander's personal bintray.

### LINKED ISSUE #189: Publish logins-api AAR
PR #178 has two parts effectively, building the AAR (including the cargo portions of the build -- e.g. from nothing), and publishing it.

Publishing it can't be finished until @ncalexan comes back from PTO, but building it can, so we merged/will merge #178 so that lockbox can do a local build without *too* much work.

### PR DESCRIPTION
**Pull request by ncalexan**
Fixes #189

### CODE REVIEW

**Comment by ncalexan** [comment]
The AAR itself is at https://dl.bintray.com/ncalexander/application-services/org/mozilla/sync15/logins/0.3.0/logins-0.3.0.aar (again, cert error).  It looks like:
```
M Filemode      Length  Date         Time      File
- ----------  --------  -----------  --------  -----------------------------------
  -rw-r--r--       304   1-Feb-1980  00:00:00  AndroidManifest.xml
  -rw-r--r--     77501   1-Feb-1980  00:00:00  R.txt
  -rw-r--r--     73871   1-Feb-1980  00:00:00  classes.jar
  drwxr-xr-x         0   1-Feb-1980  00:00:00  jni/
  drwxr-xr-x         0   1-Feb-1980  00:00:00  jni/arm64-v8a/
  -rw-r--r--  114682888   1-Feb-1980  00:00:00  jni/arm64-v8a/libloginsapi_ffi.so
  drwxr-xr-x         0   1-Feb-1980  00:00:00  jni/armeabi-v7a/
  -rw-r--r--  113297988   1-Feb-1980  00:00:00  jni/armeabi-v7a/libloginsapi_ffi.so
  drwxr-xr-x         0   1-Feb-1980  00:00:00  jni/x86/
  -rw-r--r--  122422404   1-Feb-1980  00:00:00  jni/x86/libloginsapi_ffi.so
  drwxr-xr-x         0   1-Feb-1980  00:00:00  x86/
  drwxr-xr-x         0   1-Feb-1980  00:00:00  arm64-v8a/
  drwxr-xr-x         0   1-Feb-1980  00:00:00  armeabi-v7a/
- ----------  --------  -----------  --------  -----------------------------------
              350554956                         13 files
```
I'm pretty concerned that `R.txt` has things it shouldn't have in it, but I'll try to strip that down in follow-up -- we must be (incorrectly) depending on some Android support library with resources.

**Comment by ncalexan** [comment]
Those `.so` files do have symbols, BTW -- that's why they're so large.  We can try to publish release and debug versions with different code optimization settings and strip settings if we want to do better in the future, but that can be follow-up.

File: logins-api/android/gradle.properties [hunk 1]
```diff
@@ -11,3 +11,13 @@ org.gradle.jvmargs=-Xmx1536m
...
 # org.gradle.parallel=true
+
+libGroupId=org.mozilla.sync15
+libRepositoryName=application-services
+libProjectName=application-services
+libProjectDescription=Firefox Application Services
+libUrl=https://github.com/mozilla/application-services
+libVcsUrl=https://github.com/mozilla/application-services.git
+
+libLicense 'Apache-2.0'
```
Comment (thomcc) [inline logins-api/android/gradle.properties]: Probably should be MPL given the discussion the other day.

File: .taskcluster.yml [hunk 2]
```diff
@@ -88,9 +94,11 @@ tasks:
       github:
         events:
           - tag
+    scopes:
+      - "secrets:get:project/application-services/publish"
     payload:
-      maxRunTime: 3600
-      deadline: "{{ '2 hours' | $fromNow }}"
+      maxRunTime: 7200
+      deadline: "{{ '4 hours' | $fromNow }}"
```
Comment (thomcc) [inline .taskcluster.yml]: Yikes, 4 hour build!

File: logins-api/android/gradle.properties (hunk 1 above)
Comment (ncalexan) [reply logins-api/android/gradle.properties]: Thanks for the reminder -- I will update, and perhaps I'll land #233 before this.

File: .taskcluster.yml (hunk 2 above)
Comment (ncalexan) [reply .taskcluster.yml]: Actually a 2 hour build (`maxRunTime`).  I think `deadline` is the sum of the time to schedule the job in TC and the time to actually complete, although I can't guarantee that.

**Pull request merged**